java/libphonenumber/test/com/google/i18n/phonenumbers/PhoneNumberMatcherTest.java

```diff
@@ -423,6 +423,42 @@ public void testDoesNotMatchMultiplePhoneNumbersSeparatedWithNoWhiteSpace() thro
     assertTrue(hasNoMatches(phoneUtil.findNumbers(text, region)));
   }
 
+  public void testMatchesMultiplePhoneNumbersWithExtensionSeparatedByPhoneNumberPunctuation() throws Exception {
+    String text = "Call 650-253-4561x25,455-234-3451";
+    String region = RegionCode.US;
+
+    PhoneNumber number1 = new PhoneNumber();
+    number1.setCountryCode(phoneUtil.getCountryCodeForRegion(region));
+    number1.setNationalNumber(6502534561L);
+    number1.setExtension("25");
+    PhoneNumberMatch match1 = new PhoneNumberMatch(5, "650-253-4561x25", number1);
+
+    PhoneNumber number2 = new PhoneNumber();
+    number2.setCountryCode(phoneUtil.getCountryCodeForRegion(region));
+    number2.setNationalNumber(4552343451L);
+    PhoneNumberMatch match2 = new PhoneNumberMatch(21, "455-234-3451", number2);
+
+    Iterator<PhoneNumberMatch> matches = phoneUtil.findNumbers(text, region).iterator();
+    assertEquals(match1, matches.next());
+    assertEquals(match2, matches.next());
+  }
+
+  public void testMatchesPhoneNumberWithExtensionWithExtraPunctuation() throws Exception {
+    String text = "Call 650-253-4561x25,455";
+    String region = RegionCode.US;
+
+    PhoneNumber number1 = new PhoneNumber();
+    number1.setCountryCode(phoneUtil.getCountryCodeForRegion(region));
+    number1.setNationalNumber(6502534561L);
```

Comment:
8642014971

Reply:
@GoogleJacob first time I see you comment on this project. What's your interest in / relationship with this change / pull-request?

Also, regarding this comment and the other comment, what do you mean? It's a bit cryptic. Please elaborate.

Reply:
@voronaam: Please ignore GoogleJacob's comments (unless we hear back).